widget/src/utils/routes.ts

```diff
@@ -20,36 +20,19 @@ const deduplicate = <T extends { uid?: string; id?: string }>(items: T[]): T[] =
   return Array.from(itemMap.values())
 }
 
-/** Filters all chains by available routes. */
-export const getTransferSourceChains = (): Chain[] => {
-  return MainnetRegistry.chains.filter(chain =>
-    MainnetRegistry.routes.some(route => route.from === chain.uid),
-  )
-}
-
-/** Filters all chains by selected source chain, selected token and available routes */
-export const getTransferDestinationChains = (
-  sourceChain: Chain | null,
-  token: Token | null,
-): Chain[] => {
-  if (!sourceChain || !token) return []
-
-  return MainnetRegistry.chains.filter(c =>
-    MainnetRegistry.routes.some(
-      route =>
-        route.from === sourceChain.uid && route.tokens.includes(token.id) && route.to === c.uid,
-    ),
-  )
-}
-
 /** Filters all tokens by selected source chain and available routes */
 export const getTransferTokens = (
   sourceChain: Chain | null,
   destinationChain: Chain | null,
+  allowedTokens?: Token['id'][] | undefined,
```

Comment:
```suggestion
  filteredTokens?: Token['id'][] | undefined,
```

In this context it makes sense to have the `filtered` prefix to give context to the logic.

Reply:
ah I see. No, let it be, keep it as is.